ibis/backends/impala/tests/test_patched.py

```diff
@@ -8,6 +8,7 @@ def patch_execute(con):
 
 
 def test_invalidate_metadata(con, test_data_db):
+    con = con.backend
```

Comment:
Can you instead return this from the fixture? Not a huge fan of rebinding variables of with different types of values.

Reply:
Good point, implemented this way first to see if tests were passing, and never updated. Thanks.